### ISSUE TITLE
[Thumbnail] Jellyfin

### ISSUE THREAD

**Issue by Schluggi** [andOTP/andOTP#549]
**Name**: Jellyfin
**Website**: https://jellyfin.org/
**Link to SVG or EPS image**: https://jellyfin.org/images/banner-dark.svg

**Comment by ziegenberg** [comment]
Does Jellyfin currently support 2FA? Looks like to me that feature request is still open: https://features.jellyfin.org/posts/26/add-support-for-two-factor-authentication

**Comment by ziegenberg** [comment]
I'd suggest to close this issue until 2FA is implemented in Jellyfin.
See #550 also.

**Comment by Schluggi** [comment]
Yes. 2FA is not official supported at this time. But i've added my own quick and dirty 2FA-script and want to use the right icon for this. 
Same applies to #550.

andOTP already has that "useless" kind of icons. There is no way to use andOTP for [Steam ](https://steamcommunity.com/discussions/forum/10/492379439683498898/)for example, but the icon is there.

**Comment by flocke** [comment]
Yes you can use andOTP with Steam. Check out #510 and https://github.com/andOTP/andOTP/wiki/Special-features#steam-guard-codes

**Comment by Schluggi** [comment]
Wow. That's awesome! It works fine! Thanks for the hint.  

@ziegenberg 
In this case... the battle.net-icon seems "useless" too for me.
AFAIK, you have to use the Blizzard Authenticator App. (Or i missed this feature as well 😅)

**Comment by flocke** [comment]
Battle.net works as well, but it's a little more involved getting the secret for your account: https://github.com/andOTP/andOTP/wiki/Shared-secrets#battlenet

**Comment by alexanderadam** [comment]
This icon would also be available [via SuperTinyIcons](https://github.com/edent/SuperTinyIcons#:~:text=Jellyfin) once [PR 398](https://github.com/andOTP/andOTP/pull/398) will be considered again.

**PS:** Apart from that, [the footprint of the asset size could be decreased by replacing some of the currently integrated icons with the tiny ones](https://github.com/andOTP/andOTP/issues/550#issuecomment-665121611).
